### PR TITLE
fix: correct format for g load various pages

### LINKED ISSUE #843: correct format of g load
The g for g load should be formatted in italic and without space after the number.

`grams: 1 g`
`g load: 1*g*`

should be corrected on these pages:
docs/pilots-corner/a32nx-briefing/ecam/sd/index.md
docs/pilots-corner/advanced-guides/protections/overview.md
docs/pilots-corner/advanced-guides/protections/abnormallaw.md
docs/fbw-a32nx/feature-guides/custom-hydraulics.md
docs/pilots-corner/a32nx-briefing/limitations.md

### PR DESCRIPTION
**Pull request by Valastiri**
closes: #843

## Summary
Reference issue above.

### Notes
I could not find g load references on the following pages (might be the fatigue - point me in the right direction if there are references):
- custom hydraulics
- abnormal law (protections)

### Location
- pilots-corner/a32nx-briefing/ecam/sd/index.md
- pilots-corner/advanced-guides/protections/overview.md
- pilots-corner/a32nx-briefing/limitations.md

Discord username (if different from GitHub):
 valastiri

### CODE REVIEW

File: docs/pilots-corner/advanced-guides/protections/overview.md [hunk 1]
```diff
@@ -170,15 +170,15 @@ Maneuvre Protection, also called Maneuvre Protection, enables immediate PF react
 
 The load factor limit is:
 
-- - 1.0 to + 2.5 g load factor for clean configuration
-- 0 to + 2.0 g positive load factor for other configurations
+- - 1.0 to + 2.5 *g* load factor for clean configuration
```
Comment (Sleinmaster) [inline docs/pilots-corner/advanced-guides/protections/overview.md]:
```suggestion
- - 1.0 to + 2.5*g* load factor for clean configuration
```

File: docs/pilots-corner/advanced-guides/protections/overview.md [hunk 2]
```diff
@@ -170,15 +170,15 @@ Maneuvre Protection, also called Maneuvre Protection, enables immediate PF react
...
+- 0 to + 2.0 *g* positive load factor for other configurations
```
Comment (Sleinmaster) [inline docs/pilots-corner/advanced-guides/protections/overview.md]:
```suggestion
- 0 to + 2.0*g* positive load factor for other configurations
```

**Review by Sleinmaster — COMMENTED** [review]
@Valastiri so you beat me to it :D
Looks good, just two small spaces that should be removed as well.

**Comment by Sleinmaster** [comment]
@Valastiri about the pages where you couldn't find it: that's very possible that these two pages just mentioned g loads without actual numbers. 
I just made a quick search through the GitHub repository at work. If they wouldn't have GitHub.dev blocked I'd have corrected the few occurrences straight away.

**Comment by Valastiri** [comment]
> @Valastiri about the pages where you couldn't find it: that's very possible that these two pages just mentioned g loads without actual numbers. I just made a quick search through the GitHub repository at work. If they wouldn't have GitHub.dev blocked I'd have corrected the few occurrences straight away.

no worries -- using a new IDE temporarily so a bit lost in the sauce. Thanks for the quicky review!

**Pull request merged**